query/src/test/java/org/infinispan/query/continuous/ContinuousQueryTest.java

```diff
@@ -21,6 +15,13 @@
 import org.infinispan.util.TimeService;
 import org.testng.annotations.Test;
 
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import static org.infinispan.query.dsl.Expression.max;
+import static org.infinispan.query.dsl.Expression.param;
+import static org.junit.Assert.assertEquals;
```

Comment:
Now here's an unnecessary import reordering -- the code style from https://github.com/infinispan/infinispan/pull/4439 says static imports should be on top.